apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -851,8 +852,8 @@ ttl_scenario_create_and_extend(Start0, Extend0, Stop0) ->
       || Start <- List(Start0), Extend <- List(Extend0), Stop <- List(Stop0),
          TTLo <- [ T || Extend == false, T <- ttls(Start, [Stop]) ] ++
                  [ T || Extend /= false, T <- ttls(Start, [Extend + 5]) ],
-         TTLe <- [ 0 || Extend == false ] ++
-                 [ T || Extend /= false, T <- ttls(ttl_height(Start, TTLo), [Stop]) ] ].
+         TTLe = {delta, _} <- [ 0 || Extend == false ] ++
```

Comment:
I see this filters only the ones matching `{delta, _}` hence leaving out the ones matching `{block, _}`. Ok.

Reply:
Oh, but the `[ 0 || Extend == false ]` will never match, so we won't get any test cases that doesn't extend. This should be `[ {delta, 0} || Extend == false ]` instead.